HomeTask_4.py

```diff
@@ -0,0 +1,19 @@
+"""HomeTask_4.
```

Comment:
(venv) [antoha@dlhn test_project]$ ./bin/lint ./T-HOMETASKS/L04/l04_h01_DmitriiPopov.py
./T-HOMETASKS/L04/l04_h01_DmitriiPopov.py:1:1: D205 1 blank line required between summary line and description
./T-HOMETASKS/L04/l04_h01_DmitriiPopov.py:3:80: E501 line too long (101 > 79 characters)
./T-HOMETASKS/L04/l04_h01_DmitriiPopov.py:16:38: W292 no newline at end of file